IonDotnet/Internals/SharedSymbolTable.cs

```diff
@@ -110,7 +110,8 @@ public SymbolToken Find(string text)
                 return default;
 
             var internedText = _symbolNames[sid - 1];
-            return new SymbolToken(internedText, sid);
+            // Normalize the SID and import reference to this table.
+            return new SymbolToken(internedText, sid, new ImportLocation(Name, sid));
```

Comment:
This doesn't look quite right, as the `SymbolToken`'s Sid will only be the same as the ImportLocation's Sid for the system symbol table (i.e. the first shared symbol table). In this context, the `SymbolToken`'s Sid (i.e. the local symbol table SID) would be unknown.

Reply:
Fixed per your suggestion. Please review.

Reply:
Do we make `SymbolToken`'s SID unknown because, since the text is known, returning the value of the SID would be irrelevant?

Reply:
Yes.